src/ZODB/DemoStorage.py

```diff
@@ -309,7 +309,7 @@ def store(self, oid, serial, data, version, transaction):
         if old != serial:
             rdata = self.tryToResolveConflict(oid, old, serial, data)
             self.changes.store(oid, old, rdata, '', transaction)
-            return ResolvedSerial
+            return True
```

Comment:
Disregard the comment above.  But we need to correct ``store``'s contract in IMultiCommitStorage.